08-projects.Rmd

```diff
@@ -6155,11 +6182,11 @@ A public sample of the data can be found [here](https://www.datadepot.rcac.purdu
 - List any primary or foreign keys in the `episodes` table.
 ```
 
-If you paste a `title_id` to the end of the following url, it will pull up the page for the title. For example, https://www.imdb.com/title/tt0413573 leads to the page for the TV series Grey's Anatomy. 
+If you paste a `title_id` to the end of the following url, it will pull up the page for the title. For example, https://www.imdb.com/title/tt0413573 leads to the page for the TV series _Grey's Anatomy_. 
 
-##### 3. Write a query to confirm that the `title_id` tt0413573 does indeed belong to Grey's Anatomy.
+##### 3. Write a query to confirm that the `title_id` tt0413573 does indeed belong to _Grey's Anatomy_.
 
-**Relevant topics:** [select](#sql), [where](#sql-where)
+**Relevant topics:** [SELECT](#sql-examples), [WHERE](#sql-where)
```

Comment:
The link to SELECT is changed from `#sql` to `#sql-examples` simply for consistency with 390 Project 13.

Reply:
Thanks!